js_modules/dagster-ui/packages/ui-core/src/ticks/TickStatusTag.tsx

```diff
@@ -29,15 +30,21 @@ export const TickStatusTag = ({
   const [showErrors, setShowErrors] = React.useState(false);
   const tag = React.useMemo(() => {
     const isAssetDaemonTick = 'requestedAssetMaterializationCount' in tick;
+    const requestedItem = isAssetDaemonTick ? 'asset materialization' : 'run';
     switch (tick.status) {
       case InstigationTickStatus.STARTED:
         return (
           <Tag intent="primary" icon={isStuckStarted ? undefined : 'spinner'}>
-            {isStuckStarted ? 'Started' : 'Evaluating'}
+            {isStuckStarted ? 'In progress' : 'Evaluating'}
           </Tag>
         );
       case InstigationTickStatus.SKIPPED:
-        const tag = <BaseTag fillColor={colorBackgroundLighter()} label="0 requested" />;
+        const tag = (
+          <BaseTag
+            fillColor={colorBackgroundLighter()}
+            label={`0 ${requestedItem}${ifPlural(0, '', 's')} requested`}
```

Comment:
I don't love programmatically sticking `s` on the end of nouns this way, but that's a longer term thing we need to figure out in our copy strings.

In any case, it's always zero here so you may as well do:

```
const label = isAssetDaemonTick ? '0 materializations requested' : '0 runs requested';
```